src/components/workspaces/boards-list.style.ts

```diff
@@ -6,15 +6,18 @@ export const WsSection = styled.section`
   grid-template-columns: 1fr 8fr;
   padding: 3em 0;
 `;
-export const WsNav = styled.nav`
-  h3 {
-    font-size: 1.3em;
-  }
-  ul {
-    list-style-type: none;
-  }
+export const WsNav = styled.nav``;
+
+export const WsNavHeader = styled.h3`
```

Comment:
if you intend to use this style for only the one component, I think it's better to define this styled-component in the same file as the component. Because if you export these styled-components, other members might use them from different files against your will.

Reply:
Okay. I'll do that after I refactor the styling to be responsive.